src/System.ServiceModel.Primitives/tests/Description/OperationBehaviorTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.Threading.Tasks;
+using Xunit;
+
+public static class OperationBehaviorTest
+{
+    [Fact]
+    [ActiveIssue(280)]
+    public static void IOperationBehavior_Methods_AreCalled()
+    {
+        DuplexClientBase<ICustomOperationBehaviorDuplexService> duplexService = null;
+        ICustomOperationBehaviorDuplexService proxy = null;
+
+        NetTcpBinding binding = new NetTcpBinding();
+        binding.Security.Mode = SecurityMode.None;
+
+        WcfDuplexServiceCallback callbackService = new WcfDuplexServiceCallback();
+        InstanceContext context = new InstanceContext(callbackService);
+
+        duplexService = new MyDuplexClientBase<ICustomOperationBehaviorDuplexService>(context, binding, new EndpointAddress(FakeAddress.TcpAddress));
+        proxy = duplexService.ChannelFactory.CreateChannel();
+
+        // Allow work to be done before validation
+        Task t = Task.Delay(1000);
```

Comment:
Can we do something like: 

```
int iterations = 10; 
while (iterations > 0)
{
    Task.Delay(100).Wait(); 
    if (...collection.Count > 3) { break; } 
    iterations--;
}
```

as a tight loop or something? 

I agree with Ron though, we can worry about optimizing later for runtime. LGTM